apps/api/src/app/integrations/usecases/create-integration/create-integration.usecase.ts

```diff
@@ -120,6 +136,16 @@ export class CreateIntegration {
         });
       }
 
+      if (integrationEntity.active) {
```

Comment:
Should I move this to the disable Novu integration usecase?

Reply:
I think not because this condition is related to the integration being created, not the Novu integration we want to disable.